concepts/05 UI Components/Toast/00 Getting Started with Toast/05 Display a Toast/05 Use the 'notify' Method.md

```diff
@@ -0,0 +1,265 @@
+Call the **notify** method to display a Toast. This method can accept two slightly different sets of arguments. 
```

Comment:
`two slightly different` In technical terms, something is either different or it is the same. What makes them 'slightly' different?

Reply:
I will remove the word "slightly"

Reply:
Ok